packages/lab/stories/scrim/scrim.stories.tsx

```diff
@@ -1,134 +1,155 @@
+export const CloseOnEscape: StoryFn<typeof Scrim> = () => {
   const [open, setOpen] = useState(false);
-  const parentRef = useRef(null);
 
-  const handleRequestOpen = () => {
+  const handleOpen = () => {
     setOpen(true);
   };
   const handleClose = () => {
     setOpen(false);
   };
 
-  const sectionStyle = { padding: 0, border: "none" };
-  const headerStyle = { fontSize: "1.17em" };
+  return (
+    <div style={exampleStyle}>
+      <Content />
+      <div style={buttonStyle}>
+        <Button onClick={handleOpen}>Click to open scrim</Button>
+        <Scrim
+          aria-label="Example Scrim"
+          open={open}
+          closeWithEscape
+          onClose={handleClose}
+        >
+          <Text>Press escape to close scrim</Text>
+        </Scrim>
+      </div>
+    </div>
+  );
+};
 
+export const WithSpinner: StoryFn<typeof Scrim> = () => {
   return (
-    <div
-      data-testid="skip-link-example-copy-for-scrim"
-      ref={parentRef}
-      style={{
-        position: "relative",
-        border: "1px solid grey",
-        maxWidth: 500,
-        padding: 24,
-      }}
-    >
-      <div
-        style={{
-          fontSize: 24,
-          lineHeight: 3.5,
-        }}
-      >
-        What we do
+    <div style={exampleStyle}>
+      <Content />
+      <div style={buttonStyle}>
+        <Scrim aria-label="Example Scrim" open closeWithEscape>
+          <Spinner size="medium" />
+        </Scrim>
       </div>
+    </div>
+  );
+};
+
+export const Fixed: StoryFn<typeof Scrim> = () => {
+  const [open, setOpen] = useState(false);
+
+  const handleOpen = () => {
+    setOpen(true);
+  };
+  const handleClose = () => {
+    setOpen(false);
+  };
 
-      <article id="main">
-        <section style={sectionStyle}>
-          <h1 style={headerStyle}>Investment Banking</h1>
-          <p>
-            A global leader, we deliver strategic advice and solutions,
-            including capital raising, risk management, and trade finance to
-            corporations, institutions and governments.
-          </p>
-        </section>
-
-        <section style={{ ...sectionStyle, marginTop: 20 }}>
-          <h1 style={headerStyle}>Markets</h1>
-          <p>
-            Serving the world&apos;s largest corporate clients and institutional
-            investors, we support the investment cycle with market-leading
-            research, analytics and trade execution across multiple asset
-            classes.
-          </p>
-        </section>
-      </article>
-
-      <div style={{ marginTop: 30, textAlign: "right" }}>
-        <Button onClick={handleRequestOpen}>CLICK TO OPEN SCRIM</Button>
+  return (
+    <div style={exampleStyle}>
+      <Content />
+      <div style={buttonStyle}>
+        <Button onClick={handleOpen}>Click to open scrim</Button>
+        <Scrim aria-label="Example Scrim" open={open} fixed>
+          <Button onClick={handleClose}>Click to close scrim</Button>
```

Comment:
Was this intentionally left with a Button inside the Scrim?

Reply:
yes since this example is the whole screen it can't really be anywhere else. planning to go over the examples later

Reply:
It's more that the other examples has been updated to text inside and to close when the backdrop is clicked